core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -1126,6 +1126,17 @@ wasm_instantiate(WASMModule *module, bool is_sub_inst, uint32 stack_size,
     if (!module)
         return NULL;
 
+#if WASM_ENABLE_DEBUG_INTERP != 0
+    if (!is_sub_inst) {
+        os_mutex_lock(&module->ref_count_lock);
+        if (module->ref_count != 0) {
+            LOG_WARNING("multiple instance referencing the same module may "
```

Comment:
Had better change to LOG_WARNING("warning: ...")

Reply:
multiple instance**s** referencing?

Reply:
Done